reporters/cloud-run-scripts/CloudResponse.js

```diff
@@ -14,13 +14,17 @@ const app = express();
 app.use(express.json()); // Enables json to be parsed by the express process.
 const exec = require("child_process").exec;
 
+const { Logger, waitForLogger } = require("../../financial-templates-lib/logger/Logger");
+
 app.post("/", async (req, res) => {
   try {
-    console.log("Executing GCP Cloud Run API Call");
+    Logger.debug({
+      at: "CloudRunnerResponse",
+      message: "Executing GCP Cloud Run API Call",
+      reqBody: req.body,
+      childProcessIdentifier: req.body.environmentVariables.BOT_IDENTIFIER
+    });
     if (!req.body.cloudRunCommand) {
-      res.status(400).send({
-        message: "ERROR: Body missing json cloudRunCommand!"
-      });
       throw "ERROR: Missing cloudRunCommand";
```

Comment:
The reason I did it like this is because I feed the caught error into the HTTP response sent back to the hub. You are right though that this could be a full error object with stack traces but this becomes a bit of a pain to deal with. However, for consistency sake, I agree with you it should throw a `new Error`

Reply:
If it's a pain right now, totally cool with leaving this as is. Might be worth a TODO to change it to an error object.